src/graphql/resolvers/address.resolvers.ts

```diff
@@ -1,17 +1,17 @@
-import { createAddressValidator } from "../../validations/address.validations"
+import { createAddressValidator } from '../../validations/address.validations'
 import service from '../../services/domain/address.service'
 
 export const addressResolvers = {
   Mutation: {
     async createAddress(_, { input }, context) {
       await createAddressValidator.validate(input, { abortEarly: false })
-      await service.create(input)
+      await service.create(input, context.customer.id)
       return true
     },
   },
   Query: {
-    async addresses() {
-      return service.findAll()
-    }
-  }
+    async addresses(_, __, context) {
```

Comment:
what is the using for this: (_, __,)?

Reply:
As we don't need the first 2 parameters, this is telling typescript to not give a warning for the unused vars